### ISSUE TITLE
Support RHEL

### ISSUE THREAD

**Issue by mengesb** [dnsimple/chef-pdns#93]
The current cookbook design restricts to the centos platform. Extending this to support RHEL with CentOS by changing `platform` to `platform_family in a few areas.

This will rectify the following sort of error on RedHat I believe:
```
x.x.x.x ================================================================================
x.x.x.x Recipe Compile Error in /var/chef/cache/cookbooks/wrapper_ckbk/recipes/default.rb
x.x.x.x ================================================================================
x.x.x.x
x.x.x.x Chef::Exceptions::NoSuchResourceType
x.x.x.x ------------------------------------
x.x.x.x Cannot find a resource for pdns_authoritative_install on redhat version 7.4
x.x.x.x
x.x.x.x Cookbook Trace:
x.x.x.x ---------------
x.x.x.x   /var/chef/cache/cookbooks/wrapper_ckbk/recipes/package.rb:24:in `from_file'
x.x.x.x   /var/chef/cache/cookbooks/wrapper_ckbk/recipes/default.rb:19:in `from_file'
x.x.x.x
x.x.x.x Relevant File Content:
x.x.x.x ----------------------
x.x.x.x /var/chef/cache/cookbooks/wrapper_ckbk/recipes/package.rb:
x.x.x.x
x.x.x.x  17:  # limitations under the License.
x.x.x.x  18:
x.x.x.x  19:  package 'bind-utils' do
x.x.x.x  20:    action :install
x.x.x.x  21:    only_if { node['platform_family'] == 'rhel' }
x.x.x.x  22:  end
x.x.x.x  23:
x.x.x.x  24>> pdns_authoritative_install node['wrapper_ckbk']['pdns_service'] do
x.x.x.x  25:    action node['wrapper_ckbk']['role'] == 'authoritative' ? 'install'.to_sym : 'uninstall'.to_sym
x.x.x.x  26:  end
x.x.x.x  27:
x.x.x.x  28:  pdns_recursor_install node['wrapper_ckbk']['pdns_service'] do
x.x.x.x  29:    action node['wrapper_ckbk']['role'] == 'recursor' ? 'install'.to_sym : 'uninstall'.to_sym
x.x.x.x  30:  end
x.x.x.x  31:
x.x.x.x
x.x.x.x System Info:
x.x.x.x ------------
x.x.x.x chef_version=13.8.3
x.x.x.x platform=redhat
x.x.x.x platform_version=7.4
x.x.x.x ruby=ruby 2.4.3p205 (2017-12-14 revision 61247) [x86_64-linux]
x.x.x.x program_name=chef-client worker: ppid=4447;start=22:19:03;
x.x.x.x executable=/opt/chef/bin/chef-client
x.x.x.x
x.x.x.x
x.x.x.x Running handlers:
x.x.x.x [2018-03-09T22:19:07+00:00] ERROR: Running exception handlers
x.x.x.x Running handlers complete
x.x.x.x [2018-03-09T22:19:07+00:00] ERROR: Exception handlers complete
x.x.x.x Chef Client failed. 0 resources updated in 03 seconds
x.x.x.x [2018-03-09T22:19:07+00:00] FATAL: Stacktrace dumped to /var/chef/cache/chef-stacktrace.out
x.x.x.x [2018-03-09T22:19:07+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
x.x.x.x [2018-03-09T22:19:07+00:00] ERROR: Cannot find a resource for pdns_authoritative_install on redhat version 7.4
x.x.x.x [2018-03-09T22:19:07+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
```

**Comment by onlyhavecans** [comment]
I released your patch as 4.4.0!

**Comment by mengesb** [comment]
Verified that this resolved my problems on RHEL! Thanks! 🎉
